src/ImmutablePropTypes.js

```diff
@@ -50,7 +50,7 @@ function getPropType(propValue) {
 function createChainableTypeChecker(validate) {
   function checkType(isRequired, props, propName, componentName, location) {
     componentName = componentName || ANONYMOUS;
-    if (props[propName] == null) {
+    if (props[propName] === null) {
```

Comment:
Got it. It must have just been a reflexive fix for me - my local linter always bugs me about using `===` when comparing with `null`